sandbox/src/index.js

```diff
@@ -19,8 +19,6 @@ import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(<App />, document.getElementById('root'));
 
-const promise = new Promise(resolve => resolve("done"));
-
```

Comment:
Thanks for removing this.  I added it to test if IE was working correctly with promises.